sqllogictest-bin/src/main.rs

```diff
@@ -93,6 +93,15 @@ struct Opt {
     /// Reformats the test files.
     #[clap(long)]
     format: bool,
+
+    /// Add a label for conditions.
+    ///
+    /// Records with `skipif label` will be skipped if the label is present.
+    /// Records with `onlyif label` will be executed only if the label is present.
+    ///
+    /// The engine name is a label by default.
+    #[clap(long = "label")]
+    labels: Vec<String>,
```

Comment:
Multiple labels are supported so that combinations are possible. e.g. `risingwave` + `in-mem` for this [use case](https://github.com/risingwavelabs/risingwave/pull/9013#issuecomment-1526873123)

Reply:
Why not e.g., risingwave-in-men? 🤪

Reply:
I think multi labels are more flexible. 🤪